extensions/ql-vscode/src/config.ts

```diff
@@ -211,6 +222,21 @@ export class QueryHistoryConfigListener extends ConfigListener implements QueryH
   }
 }
 
+export class CliConfigListener extends ConfigListener implements CliConfig {
+
+  public get numberTestThreads(): number {
+    return NUMBER_OF_TEST_THREADS_SETTING.getValue();
+  }
+
+  public get onDidChangeConfiguration(): Event<void> {
```

Comment:
This suggests we could push this common listener property into the superclass. But we can do that later.

Reply:
I can do that now.